### ISSUE TITLE
Chef Guard says changes are saved when they aren't

### ISSUE THREAD

**Issue by Tech356** [schubergphilis/chef-guard#65]
When updating a role that depends on a cookbook that isn't frozen in permissive mode, Chef Guard says that the changes are saved but when I look at the chef server and the github repo, the updated role is not there. 

```
$ knife role from file roles/test_production.json          
ERROR: Precondition Failed                                 
Response:                                                  
==== Cookbook Constraints errors found ====                
RUNNNING PERMISSIVE MODE: CHANGES ARE SAVED                

 - mycookbook version 0.2.0 needs to be frozen            
===========================================                
```

Either the chnages should be saved or Chef Guard should not say that they are. I am not sure which one is correct.

**Comment by svanharmelen** [comment]
Noticed this as well... Needs to be fixed.

**Comment by svanharmelen** [comment]
Fixed by PR #74
